enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -243,6 +243,55 @@ const questionId = 1; // This is the question ID you want to embed
 
 ```
 
+### Embedding a static dashboard
+
+After the SDK is configured, you can embed your dashboard using the `StaticDashboard` component.
+
+The API follows the same configuration as that of public dashboard embeddings, which you can read about (here)[https://www.metabase.com/docs/latest/questions/sharing/public-links#public-embed-parameters]
+
+
+#### Parameters
+
+- **dashboardId**: `DashboardId` (required) – The ID of the dashboard to embed.
+- **parameterQueryParams**: `Query` (required) – Query parameters for the dashboard.
+- **bordered**: `boolean` – Whether the dashboard should have a border.
+- **titled**: `boolean` – Whether the dashboard should display a title.
+- **hideDownloadButton**: `boolean | null` – Whether to hide the download button.
+- **hideParameters**: `string[] | null` – A list of parameters that will not be shown in the set of parameter filters. (More information here)[https://www.metabase.com/docs/latest/questions/sharing/public-links#filter-parameters]
+- **font**: `string | null` – The font to use.
+- **theme**: `DisplayTheme` – The display theme (e.g., "light", "night", or "transparent" ).
+  
+
+```jsx
+import React from "react";
+import { MetabaseProvider, StaticDashboard } from "@metabase/embedding-sdk-react";
+
+const config = {...}
+
+export default function App() {
+  const dashboardId = 1; // This is the dashboard ID you want to embed
+  const parameterQueryParams = {}; // Define your query parameters here
+
+  const font = "Inter"
+
+  return (
+    <MetabaseProvider config={config}>
+        <StaticDashboard 
+          dashboardId={dashboardId} 
+          parameterQueryParams={parameterQueryParams} 
+          bordered={true}
+          titled={false}
+          hideDownloadButton={null}
```

Comment:
This prop will be changed in https://www.notion.so/metabase/Make-knobs-for-downloads-consistent-between-public-links-and-static-embeds-add-PDF-download-a155ebfce35346178e657a440723b2a1 soon. The new prop is named `downloads` (boolean). It's worth checking with @albertoperdomo if we should go with the new API (I think we should)

Reply:
Yes! I find the `hideDownloadButton` name extremely annoying. So I'll use `download` externally, then we can switch internally?

Reply:
yeah, if I understand correctly that project I mentioned should have taken care of it. To be clear, I don't know _if_ we're going to change the name internally, but we're going to support `hide_download_button` for a few releases before completely removing it (couldn't find that info in the project but we've discussed elsewhere). So externally for static embeds `hide_download_button` would still be there, but maybe internally we'd change it to `downloads` I don't really know. That's the scope for that project.

Reply:
And yes I think you should use `downloads` (with an 's') externally to stay consistent, but it's worth checking that @albertoperdomo as well.